### ISSUE TITLE
Close icon in search, misses a label for accessibility

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#591]
The "X" of the search box should have a label for better accessibility
English "Reset search field"
Deutsch "Suchfeld zurücksetzen"
French: "Réinitialiser le champ de recherche"

![image](https://user-images.githubusercontent.com/7510842/32663256-94c24090-c62d-11e7-8a7e-492d90b8d343.png)

Please put this label also in the search of the guideline itself
Please update the release notes, for this

https://swiss.github.io/styleguide/de/header.html#02-header-04-site-search

**Comment by bit-pro-iew-eui** [comment]
NVDA Reader does not read the span-tag, and it's not accessible with the keyboard
